3ein39/BookEra
# Feature: Implement 'About Us' View and Endpoint

### Overview
This pull request introduces the 'About Us' feature, including a new view and corresponding API endpoint. Additionally, it includes changes to serve static files efficiently.

### Changes Made
1. **Create About Us View (Commit: a1e5fc0)**
   - Added the 'about_us.html' template to provide a user-friendly 'About Us' page.

2. **Create About Endpoint (Commit: 1cb3608)**
   - Implemented the '/about' endpoint to fetch relevant information for the 'About Us' page.

3. **Serve Static Files (Commit: d610af2)**
   - Configured static file serving to enhance the loading of assets, ensuring a seamless user experience.

### How to Test
1. Run the application and navigate to the '/about' endpoint to verify the display of the 'About Us' page.
2. Confirm that static files, such as stylesheets and images, are served correctly.

### Additional Information
- Please review the code changes and provide feedback.
- Ensure that the new features align with the project requirements and coding standards.

Fixes #<issue_number_if_applicable>
